apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeSection.tsx

```diff
@@ -93,18 +93,7 @@ const AssignContentTypeSection = (props: Props) => {
     setContentTypes(newContentTypes);
     const _parameters = { contentTypes: newContentTypes };
     mergeSdkParameters(_parameters);
-
-    // Do not allow saving an empty content type or one that was deleted
-    let hasValidContentTypes = true;
-
-    for (const contentType in newContentTypes) {
-      if (!contentType || !allContentTypeEntries.find((entry) => entry[0] === contentType)) {
-        hasValidContentTypes = false;
-        break;
-      }
-    }
-
-    onIsValidContentTypeAssignment(hasValidContentTypes);
+    onIsValidContentTypeAssignment(true);
```

Comment:
Here's another place I might leave a comment because I can see this hardcoded `true` being confusing to someone with fresh eyes on this code base. It'd be great for a line of context on _why_ we're always true here (i.e. because we always want to allow the save)

Reply:
👍 added a comment for clarity